### PR TITLE
Add source request/response definition to curator service API

### PR DESCRIPTION
**Pull request by khmoran**

### CODE REVIEW

File: verification/curator-service/api/openapi/openapi.yaml [hunk 1]
```diff
@@ -506,11 +506,52 @@ paths:
...
-          # TODO: Define Source object.
+      description: A source of line list case data.
+      type: object
+      properties:
+        name:
+          type: string
+        origin:
+          type: object
+          properties:
+            url:
```
Comment (axmb) [inline verification/curator-service/api/openapi/openapi.yaml:517]: Is `url` required?

File: verification/curator-service/api/openapi/openapi.yaml [hunk 2]
```diff
@@ -506,11 +506,52 @@ paths:
...
+                fields:
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      name:
+                        type: string
+                      regex:
+                        type: string
+            schedule:
```
Comment (axmb) [inline verification/curator-service/api/openapi/openapi.yaml:544]: Is `schedule` required within `automation`?

File: verification/curator-service/api/openapi/openapi.yaml [hunk 3]
```diff
@@ -506,11 +506,52 @@ paths:
...
+              type: object
+              properties:
+                awsRuleArn:
+                  type: string
+                  pattern: '^arn\:aws\:events\:.+\:.+\:rule\/.+'
+                awsScheduleExpression:
+                  type: string
+                  pattern: '^((rate\(((1 (minute|hour|day))|([1-9]+ (minutes|hours|days)))\))|(cron\((.+\s){5}.+\)))'
```
Comment (axmb) [inline verification/curator-service/api/openapi/openapi.yaml:552]: I missed this regex. Are we planning to leave these two in the Mongoose document as well?

Comment (khmoran) [reply verification/curator-service/api/openapi/openapi.yaml:552]: No personal preference -- I don't think it hurts but I guess it's not very DRY. WDYT?

Comment (axmb) [reply verification/curator-service/api/openapi/openapi.yaml:552]: (Chatted separately -- leaving as-is in the document for now.)

**Pull request merged**